WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasDisplayButton.lua

```diff
@@ -673,6 +768,43 @@ local methods = {
             self:Enable();
         end
     end,
+    ["SetDragging"] = function(self, draggingData)
+        if draggingData then
+            local id, parent = draggingData.id, draggingData.parent
+            self.frame:SetScript("OnClick", nil)
+            Hide_Tooltip()
+            if self.data.id == id then
+                local x, y = GetCursorPosition()
+                local scale = UIParent:GetScale()
+
+                self.downgroup:Hide()
+                self.loaded:Hide()
+                self.ungroup:Hide()
+                self.upgroup:Hide()
+                self.view:Hide()
+
+                self.frame:SetScript("OnUpdate", function(self,elapsed)
+                    self.elapsed = (self.elasped or 0) + elapsed
+                    if self.elapsed > 0.1 then
+                        Show_DropIndicator(draggingData)
+                        self.elapsed = 0
+                    end
+                end)
+
+                self.frame.isDragged = true
+                self.frame:StartMoving()
+                self.frame:ClearAllPoints()
+                self.frame:SetPoint("Center", UIParent, "BOTTOMLEFT", x/scale, y/scale)
```

Comment:
what you might try here, is to only move the frame vertically. That is, set the x position not based on the mouse cursor, but based on the list's position. No idea how much code that is, but I think that might look better visually.